modules/books.js

```diff
@@ -0,0 +1,16 @@
+export default class Library {
+  constructor() {
+    this.collection = [];
+  }
+
+  addBook(Booktitle, Bookauthor) {
+    if (Booktitle && Bookauthor) {
+      const newBook = { title: Booktitle, author: Bookauthor };
+      this.collection.push(newBook);
+    }
+  }
+
+  removeBook(index) {
+    this.collection.splice(index, 1);
+  }
```

Comment:
- [ ] Kindly use `arrow functions` for all the functions in the project to follow ES6 syntax as required 👍 

```
addBook = (bookTitle, bookAuthor) => {
}
```